app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-md-4">
+    <h1>Top Movies</h1>
+      <% count = 0 %>
+      <% @movies.each do |movie| %>
+        <% if count < 10 %>
+          <h2><%= link_to movie.name, movie_path(movie.id) %> <small>Ranked: <%= movie.rank %></small></h2>
+          <% count += 1 %>
+        <% end %>
+      <% end %>
+      <%= link_to "View More Movies", movies_path %>
+
+    </div>
+
+  <div class="col-md-4">
+    <h1>Top Books</h1>
+    <% count = 0 %>
+      <% @books.each do |book| %>
+        <% if count < 10 %>
+          <h2><%= link_to book.name, book_path(book.id) %> <small>Ranked: <%= book.rank %></small></h2>
+        <% count =+ 1%>
```

Comment:
Using `count` here is a clever way to only have 10 of each in your list.

Since you only use the first 10 items, do `@books` `@albums` and `@movies` need to contain more? I prefer to keep this kind of logic out of the views -- the view should just take whatever it's given and display it. Rather than passing _all_ movies into this view, you could instead return just 10 movies.